Mathlib/LinearAlgebra/TensorProduct.lean

```diff
@@ -945,6 +945,10 @@ def homTensorHomMap : (M →ₗ[R] P) ⊗[R] (N →ₗ[R] Q) →ₗ[R] M ⊗[R]
 
 variable {R M N P Q}
 
+def map₂ (f : M →ₗ[R] P →ₗ[R] Q) (g : N →ₗ[R] S →ₗ[R] T) :
```

Comment:
Can you add a short docstring? Thanks!

Reply:
Thanks for the reminder, didn't notice the missing doc string